### ISSUE TITLE
self-service-password won't run properly - Wrong apache config

### ISSUE THREAD

**Issue by micter59** [NoxInmortus/docker-ldap-tool-box#2]
Hello,

Thanks a lot for your work with this "generic" Dockerfile for the 3 apps.
I really appreciate, and started working with it.
I started with "service-desk" and "white-pages", and everything was ok.
But for self-service-password, I've got an error when trying to access to the application.
I've noticed that the application is not in "htdocs" inside the app folder, but the index.php is at the root inside the app folder.

I've found a solution for that, which is to modify the entrypoint.sh.
I'm not good with linux and bash, but I've managed to write this 

```
if [ "${LTB_PROJECT}" == "self-service-password" ] ; then
  rootApplication=/usr/share/${LTB_PROJECT}
else
  rootApplication=/usr/share/${LTB_PROJECT}/htdocs
fi
```

Then when echoing the 000-default.conf, I'm using the ${rootApplication}  : 

```
# Setup apache2 virtual host
if [ ! -f /etc/apache2/sites-available/000-default.conf ]; then
echo "<VirtualHost *:8080>
   ServerName ${APACHE_SERVERNAME:-$HOSTNAME}
   ServerAdmin ${APACHE_SERVERADMIN:-webmaster@localhost}

   DocumentRoot ${rootApplication}
   DirectoryIndex index.php" > /etc/apache2/sites-available/000-default.conf
```

And a few line further, again : 

```
# LDAP authentication
  if [ "${APACHE_LDAP_AUTH:-false}" == "true" ]; then
    cat >> /etc/apache2/sites-available/000-default.conf << EOF
<Directory ${rootApplication}>
```

That's all, and the container is now ok.

Michaël

**Comment by micter59** [comment]
Maybe it's not easy to understand me, so here is the new entrypoint.sh.
I was forced to rename it with .txt extension, because github won't accept to upload sh file.

[entrypoint.txt](https://github.com/NoxInmortus/docker-ldap-tool-box/files/5843370/entrypoint.txt)

**Comment by NoxInmortus** [comment]
Hello @micter59 ,

Your issue is very odd, I don't have this situation on my working containers and cannot reproduce even when creating it again bare.

Furthermore, you can see that the index.php is in the htdocs folder in the official repository : https://github.com/ltb-project/self-service-password/tree/master/htdocs

Can you paste your docker-compose file (or whatever else your using) ? Maybe you got something wrong somewhat

**Comment by micter59** [comment]
I've not the files here, but I'll post it later.

I forgot to tell that I modified your script to download from the archives of the project, and choose which version I want. In the archive, version 1.3, the index is at the root folder !! See : https://github.com/ltb-project/self-service-password/archive/v1.3.tar.gz

I did not check in the trunk in the repository, sorry.

What is strange, is that the other applications (ssp and service-desk) are ok like that !
